packages/atomic/src/components/atomic-search-box/atomic-search-box.pcss

```diff
@@ -1,2 +1,18 @@
 @import "tailwindcss/base";
 @import "tailwindcss/utilities";
+
+:host {
+  display: block;
+}
+
+.focus\:rounded-b-0:focus-within {
```

Comment:
I think I might not be understanding this comment correctly but I'm actually creating this utility method. I copied the tailwind naming structure here but there's no `focus-within:border-b-none` method inside tailwind itself. The applies come in This class is then used in the html to disable the bottom corner rounding on the searchbox when it is focused.

This is class is a combination of two of the other classes `rounded-t` and `rounded-b-none`. I'll move it to a global .pcss file however since we might reuse it for facets.